component-maps.yml

```diff
@@ -486,3 +493,10 @@ docker_container:
     docker_image:
     - deviceconnect
     release_component: true
+
+  mender-deviceconfig:
+    git:
+    - deviceconfig
+    docker_image:
+    - deviceconfig
+    release_component: true
```

Comment:
You are missing an entry under the `docker_image` top level key. These blocks always come in threes.